core/capabilities/integration_tests/workflow.go

```diff
@@ -47,6 +47,7 @@ targets:
       abi: "receive(report bytes)"
       deltaStage: %s
       schedule: %s
+      gasLimit: 500000
```

Comment:
@ettec To make this more realistic, this should come from the capability entry in the onchain registry

Reply:
sorted